samples/wrapper/strlcpy.c

```diff
@@ -0,0 +1,52 @@
+/*	$OpenBSD: strlcpy.c,v 1.16 2019/01/25 00:19:25 millert Exp $	*/
```

Comment:
If you use already existing functions' implementations, this file is not needed.